ethexe/service/src/lib.rs

```diff
@@ -716,7 +736,40 @@ impl Service {
                 }
                 event = rpc_receiver.maybe_next_some() => {
                     log::info!("Received RPC event {event:#?}");
+
+                    match event {
+                        RpcEvent::Transaction { transaction, response_sender } => {
+                            let res = tx_pool.process(transaction).map(|validated_tx| {
```

Comment:
what process exactly does?

Reply:
renamed to `validate` as previously it wasn't only about validation, but also the method pushed the event